sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingAsyncClient.java

```diff
@@ -219,28 +223,30 @@ String getEndpoint() {
     }
 
     Mono<Response<CustomFormModel>> getCustomModelWithResponse(String modelId, Context context) {
-        Objects.requireNonNull(modelId, "'modelId' cannot be null");
-
+        if (!CoreUtils.isNullOrEmpty(modelId)) {
+            throw logger.logExceptionAsError(new IllegalArgumentException("'modelId' is required and cannot"
+                + " be null or empty"));
+        }
         return service.getCustomModelWithResponseAsync(UUID.fromString(modelId), true, context)
             .onErrorMap(Utility::mapToHttpResponseExceptionIfExist)
             .map(response -> new SimpleResponse<>(response, toCustomFormModel(response.getValue())));
     }
 
     /**
-     * Get account information for all custom models.
+     * Get account information for all custom models on the form recognizer account.
      *
      * <p><strong>Code sample</strong></p>
      * {@codesnippet com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.getAccountProperties}
      *
-     * @return The account information.
+     * @return The requested account information details.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<AccountProperties> getAccountProperties() {
         return getAccountPropertiesWithResponse().flatMap(FluxUtil::toMono);
     }
 
     /**
-     * Get account information.
+     * Get account information for all custom models on the form recognizer account with an Http response.
```

Comment:
because it also returns information about the resource itself (like the capacity) I will suggest 
```suggestion
     * Get account information of the form recognizer account with an Http response.
```